packages/rocketchat-importer-csv/server.js

```diff
@@ -254,7 +298,7 @@ Importer.CSV = class ImporterCSV extends Importer.Base {
 							const creator = this.getUserFromUsername(msg.username);
 							if (creator) {
 								const msgObj = {
-									_id: `csv-${ csvChannel.id }-${ msg.ts }`,
+									_id: `csv-${ csvChannel.id }-${ roomCounter }-${ msg.ts }`,
```

Comment:
Yeah I think it is good, however you might want to include those caveats in the pull request's description so that it's easily visible to anyone who views this pull request from the change log. :+1: